libexec/bats-core/bats-exec-test

```diff
@@ -106,6 +106,15 @@ bats_exit_trap() {
       BATS_ERROR_STATUS=1
     fi
     printf 'not ok %d %s\n' "$BATS_SUITE_TEST_NUMBER" "${BATS_TEST_DESCRIPTION}${BATS_TEST_TIME}" >&3
+    if [[ -z "${TRACE_LEVEL}" || "${TRACE_LEVEL}" -gt 0 ]]; then
+      if [[ -n "$BATS_TRACE_COMMAND" ]]; then
```

Comment:
I believe printing here will omit the trace for multicommand tests like:

```
@test {
    run commandA
    run commandB
}
```

will only print for `run commandB`.

Reply:
Wow! What an oversight on my part 😅

I made the changes necessary to support this and updated the examples in the description :)